lib/langchain/utils/hash_transformer.rb

```diff
@@ -0,0 +1,25 @@
+module Langchain
+  module Utils
+    class HashTransformer
```

Comment:
I'm wondering if we should've just yanked out the code out of the `activesupport` lib? This is fine, but I was wondering if it could be more elegant.

Reply:
The code here is 100% genuine, not copied from ActiveSupport in any way. In fact It is GPT generated.